concepticondata/conceptlists/Williamson-1973-118.tsv

```diff
@@ -0,0 +1,119 @@
+ID	NUMBER	ENGLISH	FRENCH	GERMAN	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Williamson-1973-118-1	1	animal	animal	Tier	619	ANIMAL
+Williamson-1973-118-2	2	arm: hand	bras: main	Arm: Hand	2121	ARM OR HAND
+Williamson-1973-118-3	3	armpit	aisselle	Achselhöhle	1886	ARMPIT
+Williamson-1973-118-4	4	arrow	flèche	Pfeil	977	ARROW
+Williamson-1973-118-5	5	ashes	cendres	Asche	646	ASH
+Williamson-1973-118-6	6	ask (a question)	demander	fragen	1309	ASK (INQUIRE)
+Williamson-1973-118-7	7	bad, be	mauvais (être)	schlecht sein	1292	BAD
+Williamson-1973-118-8	8	bean	haricot	Bohne	832	BEAN
+Williamson-1973-118-9	9	beg (=ask for something)	demander (qqc à qqn)	bitten um, erbitten, betteln	1828	ASK (REQUEST)
+Williamson-1973-118-10	10	belly	ventre	Bauch, Unterleib	1251	BELLY
+Williamson-1973-118-11	11	bite	mordre	beissen	1403	BITE
+Williamson-1973-118-12	12	buffalo	buffle	Büffel	934	BUFFALO
+Williamson-1973-118-13	13	build	bâtir	bauen	1840	BUILD
+Williamson-1973-118-14	14	calabash	calebasse	Kalebasse	411	GOURD
+Williamson-1973-118-15	15	carve	sculpter	schnitzen	228	CARVE
+Williamson-1973-118-16	16	charcoal (coal)	charbon	Holzkohle (Kohle)	2659	CHARCOAL OR COAL
+Williamson-1973-118-17	17	child	enfant	Kind	2099	CHILD
+Williamson-1973-118-18	18	come out	sortir	herauskommen	1107	GO OUT
+Williamson-1973-118-19	19	cool, be(come)	se refraîchir	kühl sein, kühl werden		
+Williamson-1973-118-20	20	corpse	cadavre	Leichnam, Leiche, der Tote	767	CORPSE
+Williamson-1973-118-21	21	count: draw a line:: reckon	compter: tirer un ligne:: calculer	zählen: eine Linie ziehen: rechnen		
+Williamson-1973-118-22	22	cow	vache	Kuh	1007	COW
+Williamson-1973-118-23	23	crab	crabe	Krabbe	2348	CRAB
+Williamson-1973-118-24	24	crocodile	crocodile	Krokodil	1857	CROCODILE
+Williamson-1973-118-25	25	dance v. and n.	danser: danse	tanzen: Tanz		
+Williamson-1973-118-26	26	depth (of water)	profondeur (de l’eau)	Tiefe (vom Wasser)	2534	DEPTH
+Williamson-1973-118-27	27	die: death	mourir: mort	sterben: Tod	3186	BE DEAD OR DEATH OR DIE
```

Comment:
Would it be too pedantic if I'd say, I'd unmap this one? :) There's just one list mapped to BE DEAD OR DEATH OR DIE so far and it involves all the three components of this concept, unlike here.

Reply:
That's for the other reviewers to decide ;) In my book 3186 is a catch-all concept for languages who do not necessarily make a distinction between all the possible word classes involved here (or between state and process), but I'm not sure whether all of them need to be included or not.

Reply:
I think unmapping here for now is good. I'll make a note of this concept class (2101/3185/3186) and we can have a look at them separately.